draft-irtf-cfrg-frost.md

```diff
@@ -129,10 +129,8 @@ The following notation and terminology are used throughout this document.
 * `THRESHOLD_LIMIT` denotes the threshold number of participants required to issue a signature. More specifically,
 at least THRESHOLD_LIMIT shares must be combined to issue a valid signature.
 * `len(x)` is the length of integer input `x` as an 8-byte, big-endian integer.
-* `I2OSP(x, w)`: Convert non-negative integer `x` to a `w`-length,
-  big-endian byte string as described in {{!RFC8017}}.
-* `OS2IP(s)`: Convert byte string `s` to a non-negative integer as
-  described in {{!RFC8017}}, assuming big-endian byte order.
+* `encode_uint16(x)`: Convert two byte unsigned integer (uint16) `x` to a 2-byte,
```

Comment:
Does this need to be defined further in the spec or link to such a description?

Reply:
Is it not clearly defined already as-is?

Reply:
In the POC code it is, in the spec pseudocode it's a bit fuzzy

Reply:
Can you elaborate on how it's fuzzy, or offer a suggestion to clarify?

Reply:
Resolved, thanks!